05_stringer/CameronSchafer/main_test.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// main() test
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := "127.0.0.1\n"
+	actual := buf.String()
+
```

Comment:
Not exactly sure, but I imagine there might be a little bug; once you set `out = &buf` it is never set back to `os.Stdout`, so any tests after this one won't be able to output in the standard output. 
I guess `out = os.Stdout` would fix it

Reply:
Wait no, this only impacts the main function because it's the only place `out` is used

Reply:
Nice!